### PR TITLE
clarification on SDID as a local identifer

### PR DESCRIPTION
**Pull request by rsahita**
to address issue #76

### CODE REVIEW

File: chapter3.adoc [hunk 1]
```diff
@@ -6,7 +6,13 @@
 under which a hart is operating. The interface consists of M-mode CSRs `msdcfg`
 and `mttp`. The SDID programmed via this interface is a local identifier for the
 hart and may be used to tag hart-local resources to access-control data
-associated with the supervisor domain.
+associated with the supervisor domain. For most typical usages, the SDID will
+remain stable over the life of the supervisor domain. However, the SDID can be
+changed at the cost of fencing to flush any prior cached state. The SDID is
+unique only within a cpu-complex, hence, SDID must not be used to tag downstream
```
Comment (ved-rivos) [inline chapter3.adoc:12]: unique only to a hart.

Comment (rsahita) [inline chapter3.adoc:12]:
```suggestion
unique only to a hart, hence, SDID must not be used to tag downstream
```

File: chapter3.adoc [hunk 2]
```diff
@@ -6,7 +6,13 @@
...
 and `mttp`. The SDID programmed via this interface is a local identifier for the
 hart and may be used to tag hart-local resources to access-control data
-associated with the supervisor domain.
+associated with the supervisor domain. For most typical usages, the SDID will
+remain stable over the life of the supervisor domain. However, the SDID can be
+changed at the cost of fencing to flush any prior cached state. The SDID is
+unique only to a hart, hence, SDID must not be used to tag downstream
+transactions that are globally visible - implementations may map the SDID from
+the initiator to other (global) identifiers but that mapping is
+implementation-specific and out of scope of this specification.
```
Comment (SiFiveHolland) [inline chapter3.adoc:15]: If statement 2 from #76 is correct, then we shouldn't suggest mapping the SDID to anything outside the hart:

```suggestion
transactions that are globally visible.
```

Comment (ved-rivos) [inline chapter3.adoc:15]: We could remove the "implementations may ...." - don't need to suggest an implementation specific thing. Implementations are of course free to do whatever they need but the architecture may not want to encourage that.

**Review by SiFiveHolland — APPROVED** [review]

**Review by ved-rivos — APPROVED** [review]

**Pull request merged**